doc/en/doctest.rst

```diff
@@ -26,9 +28,11 @@ can be given multiple times in the command-line.
 
 You can also trigger running of doctests
 from docstrings in all python modules (including regular
-python test modules)::
+python test modules):
+
+.. code-block:: bash
 
-    pytest --doctest-modules
+    $ pytest --doctest-modules
```

Comment:
IIRC this will trigger a `regendoc` run, is that correct @RonnyPfannschmidt?

Reply:
yup

Reply:
@gyermolenko in that case is better to not add `$` on entries which don't have that already.

Other than that please go ahead, the other changes look good to me! 👍

Reply:
@nicoddemus do you mean 
a) `$`s should be handled in separate PR; 
or
 b) it is ok "as is"?

Reply:
Is OK as is, without the `$`. 👍 

We could probably change regendoc so `$` blocks would be opt-in instead of always being regenerated, not sure. Anyway this is a topic for another time.